src/overlays/actors/ovl_En_Ishi/z_en_ishi.h

```diff
@@ -6,26 +6,40 @@
 struct EnIshi;
 
 typedef void (*EnIshiActionFunc)(struct EnIshi*, PlayState*);
-typedef void (*EnIshiUnkFunc)(struct EnIshi*, PlayState*);
-typedef void (*EnIshiUnkFunc2)(Actor*, PlayState*);
-
-#define ENISHI_GET_1(thisx) ((thisx)->params & 1)
-#define ENISHI_GET_2(thisx) (((thisx)->params >> 1) & 1)
-#define ENISHI_GET_4(thisx) (((thisx)->params >> 2) & 1)
-#define ENISHI_GET_8(thisx) (((thisx)->params >> 3) & 1)
-#define ENISHI_GET_70(thisx) (((thisx)->params >> 4) & 7)
-#define ENISHI_GET_F0(thisx) (((thisx)->params >> 4) & 0xF)
-#define ENISHI_GET_100(thisx) (((thisx)->params >> 8) & 1)
-#define ENISHI_GET_FLAG(thisx) (((thisx)->params >> 9) & 0x7F)
+// used for dust spawn and draw funcs
+typedef void (*EnIshiMultiFunc)(struct EnIshi*, PlayState*);
+typedef void (*EnIshiSpawnDebrisFunc)(Actor*, PlayState*);
+
+// silver boulder instead of small rock
+#define ENISHI_GET_BIG_FLAG(thisx) ((thisx)->params & 1)
+#define ENISHI_GET_SPAWN_BUGS_FLAG(thisx) (((thisx)->params >> 1) & 1)
+// for sitting still on walls in TF/Ikana Graveyard
+#define ENISHI_GET_IGNORE_SNAP_TO_FLOOR(thisx) (((thisx)->params >> 2) & 1)
```

Comment:
Maybe `Float/Levitate` instead of `IGNORE_SNAP_TO_FLOOR`?

Reply:
I'm fine with this, because it describes the behavior, but adding a comment that its used to stick the rocks to static cliffs, and not actually used for levitation